src/_tsd.d.ts

```diff
@@ -1,3 +1,3 @@
 /// <reference path="../typings/knockout/knockout.d.ts" />
-/// <reference path="../node_modules/definition-header/dist/index.d.ts" />
+/// <reference path="../node_modules/tsd/node_modules/definition-header/dist/index.d.ts" />
```

Comment:
Is it a manual change?

I said because this file `_tsd.d.ts` is being generated automatically, so if it is a manual change it will be broken automatically.